src/tweak.h

```diff
@@ -176,6 +176,14 @@ class CTweakRef : public CTweakBase
         std::lock_guard<std::mutex> lck(cs_tweak);
         return *value;
     }
+
+    CTweakRef &operator=(const DataType &d)
+    {
+        std::lock_guard<std::mutex> lck(cs_tweak);
+        if (value)
+            *value = d;
+        return *this;
+    }
```

Comment:
what's the purpose of the above change?

Reply:
allows code to set the tweak value.  Previously the member variable was public.  When it was moved to be private, a getter was implemented but not a setter.